gam_g4/gam_g4/gam_lib/GamSourceManager.cpp

```diff
@@ -192,6 +192,7 @@ void GamSourceManager::GeneratePrimaries(G4Event *event) {
 void GamSourceManager::InitializeVisualization() {
     if (!fVisualizationFlag) return;
     char *argv[1]; // ok on osx
+    argv[0] = "f";
     //char **argv = new char*[1]; // not ok on osx
```

Comment:
Thanks for pointing me here. When I compile a minimal example version of this, I get a `warning: ISO C++ forbids converting a string constant to ‘char*’ [-Wwrite-strings]`. No idea whether this is related to the reported issue on Mac. Could you please try the following snipped from #27:
```suggestion
    char argv0[] = "f";
    char *argv[1] = {argv0};
```